manifests/agent.pp

```diff
@@ -12,6 +12,8 @@
   $config_group                 = 'root',
   $config_mode                  = '0644',
   $env                          = $::env,
+  $puppet_run_interval          = '1h',
```

Comment:
This would change the default behavior from 30 minutes to 1 hour. A config option should use the default behavior in accordance with the principal of least surprise.